BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+## @ GenerateKeys.py
+# Generate RSA Keys
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+##
+# Import Modules
+#
+import os
+import sys
+import re
+import glob
+import struct
+import shutil
+import subprocess
+import datetime
+import zipfile
+import ntpath
+import argparse
+import binascii
+from ctypes import *
+from CommonUtility import *
+
+
+RSA_KEY_SIZE = {
+        "RSA2048"      : '2048',
+        "RSA3072"      : '3072',
+    }
+
+
+def signing_keys():
+    signing_keys_list = []
+    signing_keys_list.append ([
+      # Key Name                    | Key File Name start |
+      # ===========================================================
+        ("MASTER_KEY",              "MasterTestKey"),

```

Comment:
Before each key, can we add a line of comment ..describing the purpose of each key w.r.t Slimbootloader, such as,
#MASTER_KEY is used for signing SBL Key Manifest.
 etc